gateway/rdb/src/main/kotlin/org/doorip/gateway/rdb/trip/ParticipantJpaEntity.kt

```
@@ -0,0 +1,24 @@
+@Table(name = "participant")
+@Entity
+internal class ParticipantJpaEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "participant_id", columnDefinition = "bigint", nullable = false)
+    var id: Long = 0
+        protected set
+
+    @Embedded
+    lateinit var propensityTag: PropensityTag
+        protected set
```

detekt (1.23.7)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)